src/ReactiveUI.Testing/ReactiveUI.Testing.csproj

```diff
@@ -4,9 +4,10 @@
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);net461;uap10.0.16299;netcoreapp3.0</TargetFrameworks>
     <AssemblyName>ReactiveUI.Testing</AssemblyName>
     <RootNamespace>ReactiveUI.Testing</RootNamespace>
-    <Description>A library to aid in writing unit tests for ReactiveUI projects</Description>
+    <PackageDescription> A MVVM framework that integrates with the Reactive Extensions for .NET to create elegant, testable User Interfaces that run on any mobile or desktop platform. This is the base package with the base platform implementations</PackageDescription>
```

Comment:
This one should probably change to mention it contains the testing tools for rxui. Probably the tags also.

Reply:
Done 😃